src/view/ChewingEditor.cpp

```diff
@@ -104,6 +104,8 @@ void ChewingEditor::execFileDialog(DialogType type)
 
 void ChewingEditor::selectImportFile()
 {
+    import_count = 0;
```

Comment:
Hide the implementation information by rewriting as following:
```c
static int import_count = 0;
...
if(import_count++ == 1) ...
```
Then, you don't have to expose it in the class declaration.